nexus/tests/output/nexus_tags.txt

```diff
@@ -173,13 +173,20 @@ silo_view                                /system/silos/{silo_name}
 silo_view_by_id                          /system/by-id/silos/{id}
 sled_list                                /system/hardware/sleds
 sled_view                                /system/hardware/sleds/{sled_id}
+system_component_version_list            /v1/system/update/components
 system_image_create                      /system/images
 system_image_delete                      /system/images/{image_name}
 system_image_list                        /system/images
 system_image_view                        /system/images/{image_name}
 system_image_view_by_id                  /system/by-id/images/{id}
+system_update_components_list            /v1/system/update/updates/{update_id}/components
+system_update_list                       /v1/system/update/updates
+system_update_start                      /v1/system/update/updates/{update_id}/start
+system_update_stop                       /v1/system/update/updates/{update_id}/stop
+system_update_view                       /v1/system/update/updates/{update_id}
 system_user_list                         /system/user
 system_user_view                         /system/user/{user_name}
+system_version                           /v1/system/update/version
```

Comment:
IF (big IF) we don't have anything else hanging off of `/system/update` other than what's here, seems like we could get away without doing my goofy `/update/updates` idea if we put system version and component version tree under `/system/version`. So:

```
system_update_list                       /v1/system/updates
system_update_view                       /v1/system/updates/{update_id}
system_update_components_list            /v1/system/updates/{update_id}/components
system_update_start                      /v1/system/updates/{update_id}/start
system_update_stop                       /v1/system/updates/{update_id}/stop

system_version                           /v1/system/version
system_component_version_list            /v1/system/version/components
```

This has the disadvantage of putting version under a different top-level key, which makes it less obvious how closely tied it is to the `/system/updates` endpoints.

Reply:
I much prefer this. The `/system/update/updates` pattern is fairly different than the rest of our API where typically resources are plural.

Reply:
I forgot, we already have one:

https://github.com/oxidecomputer/omicron/blob/75bd90550e63940bbed7ef40ee21e18fa1b628dc/nexus/src/external_api/http_entrypoints.rs#L5032-L5040

Reply:
I don't guess dropshot would let you register `/system/updates/refresh` before `/system/updates/{update_id}` to have it automatically be handled, would it? 😅. There's actually no overlap in this case because `refresh` isn't a valid UUID. If that worked, it'd be sort of ideal. 

Beyond that, I'm not really sure. I obviously don't think we should stall on this API decision and I'm fine with moving forward with what's here. Hopefully we'll figure out a better way to do this at some point and update accordingly.

Reply:
No, Dropshot would have to be modified: https://github.com/oxidecomputer/dropshot/issues/199

And more and more I agree with the comments on that issue arguing it’s a footgun.